migrations/sql/006-rename-eventtype.down.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE events SET type=feature-revive WHERE type=feature-revived;
+UPDATE events SET type=feature-archive WHERE type=feature-archived;
```

Comment:
I have not run db-migrate up yet. I didn't have postgres locally but I will install http://postgresapp.com/ as you recommended =)